ingestion/src/metadata/ingestion/source/database/common_db_source.py

```diff
@@ -389,58 +383,6 @@ def yield_table(
             logger.warning(f"Unexpected exception to yield table [{table_name}]: {exc}")
             self.status.failures.append(f"{self.config.serviceName}.{table_name}")
 
-    def yield_view_lineage(self) -> Optional[Iterable[AddLineageRequest]]:
-        logger.info("Processing Lineage for Views")
-        for view in self.context.table_views:
```

Comment:
Should we remove the `table_views` from the context?

Reply:
good catch. Thank you, removed it